test/sql-tap/gh-6024-funcs-return-bin.c

```diff
@@ -3,6 +3,12 @@
 #include "uuid/mp_uuid.h"
 #include "mp_decimal.h"
 
+char *
+tnt_mp_encode_uuid(char *data, const struct tt_uuid *uuid);
+
+char *
+tnt_mp_encode_decimal(char *data, const decimal_t *dec);
```

Comment:
Nit: I would include a header instead of copy-pasting the declarations.

Reply:
I would also leave several words in the header file about purpose of the wrapping.

Reply:
Done.